dali/operators/reader/video_reader_op.cc

```diff
@@ -26,63 +26,83 @@ namespace dali {
+
+This option is mutually exclusive with ``filenames`` and ``file_root``.)code",
       std::vector<std::string>{})
   .AddOptionalArg("file_root",
-      R"code(Path to a directory containing data files.
-This option is mutually exclusive with `filenames` and `file_list`.)code",
+      R"code(Path to a directory that contains the data files.
+
+This option is mutually exclusive with ``filenames`` and ``file_list``.)code",
       std::string())
   .AddOptionalArg("file_list",
-      R"code(Path to the file with a list of pairs ``file label``.
-This option is mutually exclusive with `filenames` and `file_root`.)code",
+      R"code(Path to the file with a list of the file label pairs.
+
+This option is mutually exclusive with ``filenames`` and ``file_root``.)code",
       std::string())
   .AddOptionalArg("enable_frame_num",
-      R"code(Return frame number output if file_list or file_root argument is passed)code",
+      R"code(If the ``file_list`` or ``file_root`` argument is passed, returns
+the frame number output.)code",
       false)
   .AddOptionalArg("enable_timestamps",
-      R"code(Return timestamps output if file_list or file_root argument is passed)code",
+      R"code(If the`` file_list`` or ``file_root`` arguments are passed, returns
+the timestamps output. )code",
       false)
   .AddArg("sequence_length",
       R"code(Frames to load per sequence.)code",
       DALI_INT32)
   .AddOptionalArg("step",
-      R"code(Frame interval between each sequence (if `step` < 0, `step` is set to `sequence_length`).)code",
+      R"code(Frame interval between each sequence.
+
+When the value is less than 0, ``step`` is set to ``sequence_length``.)code",
       -1)
   .AddOptionalArg("channels",
       R"code(Number of channels.)code",
       3)
   .AddOptionalArg("additional_decode_surfaces",
       R"code(Additional decode surfaces to use beyond minimum required.
-This is ignored when decoder is not able to determine minimum
-number of decode surfaces, which may happen when using an older driver.
-This parameter can be used trade off memory usage with performance.)code",
+
+This argument is ignored when the decoder cannot determine the minimum number of
+decode surfaces
+
+.. note::
+
+  This can happen when the driver is an older version.
+
+This parameter can be used to trade off memory usage with performance.)code",
       2)
   .AddOptionalArg("normalized",
-      R"code(Get output as normalized data.)code",
+      R"code(Gets the output as normalized data.)code",
       false)
   .AddOptionalArg("image_type",
-      R"code(The color space of the output frames (supports RGB and YCbCr).)code",
+      R"code(The color space of the output frames, and RGB and YCbCr are supported.)code",
       DALI_RGB)
   .AddOptionalArg("dtype",
-      R"code(The data type of the output frames (supports FLOAT and UINT8).)code",
+      R"code(The data type of the output frames, and  FLOAT and UINT8 are supported.)code",
```

Comment:
```suggestion
      R"(The data type of the output frames (UINT8 or FLOAT).)",
```
Really this sounds like "float" and "uint8" were supported in addition to the output frame type.

Reply:
Done